sdk/inc/azure/iot/az_iot_common.h

```diff
@@ -63,27 +63,31 @@ typedef enum
   AZ_IOT_STATUS_TIMEOUT = 504,
 } az_iot_status;
 
-/**
+/*
  *
  * Properties APIs
  *
  *   IoT message properties are used for Device-to-Cloud (D2C) as well as Cloud-to-Device (C2D).
  *   Properties are always appended to the MQTT topic of the published or received message and
- *   must contain Uri-encoded keys and values.
+ *   must contain percent-encoded names and values.
  */
```

Comment:
Are these doc comments not meant for doxygen/public?

Reply:
doxygen doesn't render these because it isn't attached to anything. It also doesn't render the documentation linearly so it's not as if it would place this together with the "properties" API. Doxygen groups by enum, structs, functions, etc. So I removed the star to make it more apparent this isn't for doxygen.